src/connection/pvws.test.ts

```diff
@@ -0,0 +1,97 @@
+import { PvwsPlugin } from "./pvws";
+import WS from "vitest-websocket-mock";
+import { DType } from "../types/dtypes";
+
+describe("PvwsPlugin", (): void => {
+  let cp: PvwsPlugin;
+  let mockConnUpdate: jest.Mock;
+  let mockValUpdate: jest.Mock;
+  let ws: WS;
+  beforeEach(async () => {
+    ws = new WS("ws://a.b.c:100/pvws/pv");
+    cp = new PvwsPlugin("a.b.c:100", false);
+    mockConnUpdate = jest.fn();
```

Comment:
Is there a reason we use jest.fn() to mock here, instead of vi.fn()?

Reply:
Ah no, I think I just missed that. Thanks for catching that - I'll fix and retest.